test/environmentSettings.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import path from 'path';
+
+export class EnvironmentSettings {
+  private static _instance: EnvironmentSettings;
+
+  private _devHubAliasName = 'vscodeOrg';
+  private _devHubUserName = 'svc_idee_bot@salesforce.com';
+  private _extensionPath = path.join(__dirname, '..', '..', 'salesforcedx-vscode', 'packages');
+  private _throttleFactor = 1;
+
+  private constructor() {
+  }
+
+  public static getInstance(): EnvironmentSettings {
+    if (!EnvironmentSettings._instance) {
+      EnvironmentSettings._instance = new EnvironmentSettings();
+
+      EnvironmentSettings._instance._devHubAliasName = process.env.DEV_HUB_ALIAS_NAME || EnvironmentSettings._instance._devHubAliasName;
+      EnvironmentSettings._instance._devHubUserName = process.env.DEV_HUB_USER_NAME || EnvironmentSettings._instance._devHubUserName;
+      EnvironmentSettings._instance._extensionPath = process.env.EXTENSION_PATH || EnvironmentSettings._instance._extensionPath;
```

Comment:
New environment variable (`EXTENSION_PATH`) was added.  The default is `{cwd}/../../salesforcedx-vscode/packages`, and if both repos are at the same location (`{location}/salesforcedx-vscode` and `{location}/salesforcedx-vscode-automation-tests`) no changes are needed.

Reply:
done